assets/dev/js/frontend/handlers/section/stretched-section.js

```diff
@@ -7,10 +7,19 @@ export default class StretchedSection extends elementorModules.frontend.handlers
 		elementorFrontend.addListenerOnce( handlerID, 'sticky:stick', this.stretch, this.$element );
 
 		elementorFrontend.addListenerOnce( handlerID, 'sticky:unstick', this.stretch, this.$element );
+
+		if ( elementorFrontend.isEditMode() ) {
+			this.onKitChangeStretchContainerChange = this.onKitChangeStretchContainerChange.bind( this );
+			elementor.channels.editor.on( 'kit:change:stretchContainer', this.onKitChangeStretchContainerChange );
```

Comment:
We should supply a general alternative for the `onGeneralSettingsChange` method

Reply:
The frontend handlers shouldn't know about the editor at all. it's a temporary handling, until we will create a Preview Handler solution.

Reply:
But you still calling the editor and listening to its events. Therefore it should be handled.